views/index.ejs

```diff
@@ -1,9 +1,10 @@
 <%- include('partials/head'); %>
 <div class="mdl-layout mdl-js-layout mdl-layout--no-desktop-drawer-button">
-    <%- include('partials/header'); %>
-    <main class="mdl-layout__content">
-        <div class="mdl-typography--text-center clemson-esports">
```

Comment:
I HATE THIS

Reply:
what happened lol it looks the same except for maybe a tab or space